geomstats/geometry/grassmannian.py

```diff
@@ -4,9 +4,11 @@
 where p <= n
 """
 
+import geomstats.backend as gs
+
 from geomstats.geometry.embedded_manifold import EmbeddedManifold
 from geomstats.geometry.euclidean_space import EuclideanMetric
-from geomstats.geometry.matrices_space import MatricesSpace
+from geomstats.geometry.matrices_space import MatricesSpace as Mat
```

Comment:
No renaming of the classes, otherwise it's very difficult for other developers to understand what is referred to.